Library/src/Systems/Assets/AssetCollection.cpp

```diff
@@ -93,12 +94,10 @@ void PrototypeDtoToAssetCollection(const chs::PrototypeDto& Dto, csp::systems::A
 
 	if (Dto.HasGroupIds())
 	{
-		auto& GroupIds			 = Dto.GetGroupIds();
-		AssetCollection.SpaceIds = csp::common::Array<csp::common::String>(GroupIds.size());
-
-		for (int i = 0; i < GroupIds.size(); ++i)
+		auto& GroupIds = Dto.GetGroupIds();
+		if (GroupIds.size() > 0)
```

Comment:
Can we get empty lines before and after scope blocks, please?

Reply:
Addressed in latest